spec/export/index.md

```diff
@@ -179,6 +181,76 @@ Note: When requesting status, the client SHOULD use an ```Accept``` header for i
 
 `GET [polling content location]`
 
+**Responses**
+<table class="table">
+  <thead>
+    <th>Status</th>
+    <th>Description</th>
+    <th>Example Output Response</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td><a href="#response---in-progress-status"><code>In-Progress</code></a><br/><span class="label label-success">202 Accepted</span></td>
+      <td>This response is returned by the server when the server is still processing the $export request.</td>
+      <td>
+        ```
+          In-progress, 85% complete
```

Comment:
This is a header, rather than a body; need to clarify this, by including the header name and explicitly mentioning that the body is empty.

Also this quoting doesn't work.

Reply:
fixed https://github.com/HL7/bulk-data/pull/9/files#diff-985cef1e2f9cd85436f27813aa6de763